src/plugin.ts

```diff
@@ -191,6 +198,30 @@ export default class DocgenPlugin implements webpack.WebpackPluginInstance {
 
             const nameForCondition = module.nameForCondition() || "";
 
+            // Ignore already built modules for webpack 5
+            if (!compilation.builtModules.has(module)) {
```

Comment:
@bebraw Hmm, in PR description you said that `It's important not to process already built modules`, but there we return early for un-build modules. Am I missing something?

Reply:
Here's the related API refactor: https://github.com/webpack/webpack/pull/7933 .

The interesting thing is that the API isn't used much. Here's the usage in webpack itself: https://github.com/webpack/webpack/search?q=builtModules&type= .

Webpack API is confusing here as it's not well documented what exactly happens on a recompilation.